targets/riscv.ld

```diff
@@ -18,7 +18,7 @@ SECTIONS
      * See: http://blog.japaric.io/stack-overflow-protection/ */
     .stack (NOLOAD) :
     {
-        . = ALIGN(4);
+        . = ALIGN(8);
```

Comment:
This is a change for all chips but probably fine.

Reply:
Yes I know, according to the RISC-V ABI the stack should even be 16 bytes aligned but since none of the chips currently supported by TinyGo have the Q extension I think we can settle with an alignment of 8 bytes. (more details [here](https://github.com/riscv/riscv-elf-psabi-doc/issues/21))

Reply:
Is this still necessary now that the K210 has a separate linker script?

Reply:
Not on 32-Bit systems no. Should I change it back to 4 Bytes or should we still respect the default stack alignment of 16 Bytes even if it works with 4 Bytes?

Reply:
Ok, for this line (the stack) it makes sense to do the alignment. But for the other alignments below I think they should be reverted back to 4.

Reply:
done